src/fmu/config/utilities.py

```diff
@@ -25,7 +25,7 @@ def yaml_load(filename, safe=True, tool=None):
     """
 
     if not os.path.isfile(filename):
-        raise IOError("File {} cannot be read".format(filename))
+        raise FileNotFoundError(filename)
```

Comment:
There is probably no point in raising this exception as such, probably exactly the same thing will happen in line 30 anyways.

Reply:
Yes, I agree. I think the entire check can be removed, as yaml will raise FileNotFoundError by itself

Reply:
Yes, that was obvious (doh!). A bit quick on the trigger, trying to take this small issue in addition to another fix. Have removed the whole thing now.